os/linux/appimage-builder.yml

```diff
@@ -0,0 +1,94 @@
+version: 1
+
+script: |
+    which unzip || apt-get update ; apt-get install wget unzip -y
+    mkdir -p AppDir/usr/share/icons/hicolor/scalable/apps
+    cp ../media/openttd.svg AppDir/usr/share/icons/hicolor/scalable/apps
+    wget -qcO opengfx.zip https://cdn.openttd.org/opengfx-releases/0.6.0/opengfx-0.6.0-all.zip
+    unzip opengfx.zip -d AppDir/usr/share/games/openttd/baseset
+    wget -qcO openmsx.zip https://cdn.openttd.org/openmsx-releases/0.3.1/openmsx-0.3.1-all.zip
+    unzip openmsx.zip -d AppDir/usr/share/games/openttd/baseset
+    wget -qcO opensfx.zip https://cdn.openttd.org/opensfx-releases/0.2.3/opensfx-0.2.3-all.zip
+    unzip opensfx.zip -d AppDir/usr/share/games/openttd/baseset
+
+AppDir:
+  path: ./AppDir
+
+  app_info:
+    id: org.openttd.openttd
+    name: openttd
+    icon: openttd
+    version: latest
+    exec: usr/bin/openttd
+
+  apt:
+    arch: amd64
+    sources:
+      - sourceline: 'deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ focal main restricted universe multiverse'
+      - sourceline: 'deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ focal-updates main restricted universe multiverse'
+      - sourceline: 'deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ focal-backports main restricted universe multiverse'
+      - sourceline: 'deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ focal-security main restricted universe multiverse'
+        key_url: 'http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x3b4fe6acc0b21f32'
+
+    include:
+      - freepats
+      - timidity
+      - libicu66
+      - liblzma5
+      - liblzo2-2
+      - libpng16-16
+      - libsdl1.2debian
+      - libxdg-basedir1
+      - libsdl2-2.0-0
+      - zlib1g
+    exclude:
+      - humanity-icon-theme
+      - adwaita-icon-theme
+      - python3
+      - python3.6
+      - python3-minimal
+      - python3.6-minimal
+      - perl
+      - perl-base
+      - pkg-config
+      - patch
+      - make
+      - dpkg-dev
+
+  after_bundle: |
+    echo "source /etc/timidity/freepats.cfg" | tee AppDir/etc/timidity/timidity.cfg
+
+  runtime:
+    path_mappings:
+      - /usr/share/games/openttd:$APPDIR/usr/share/games/openttd
+      - /etc/timidity:$APPDIR/etc/timidity
+      - /usr/lib/timidity:$APPDIR/usr/lib/timidity
+
+  test:
+    fedora:
+      image: appimagecrafters/tests-env:fedora-30
+      command: ./AppRun
+      use_host_x: true
+    debian:
+      image: appimagecrafters/tests-env:debian-stable
+      command: ./AppRun
+      use_host_x: true
+    arch:
+      image: appimagecrafters/tests-env:archlinux-latest
+      command: ./AppRun
+      use_host_x: true
+    centos:
+      image: appimagecrafters/tests-env:centos-7
+      command: ./AppRun
+      use_host_x: true
+    ubuntu:
+      image: appimagecrafters/tests-env:ubuntu-xenial
+      command: ./AppRun
+      use_host_x: true
+
+AppImage:
+  update-information: "gh-releases-zsync|OpenTTD|OpenTTD|latest|*x86_64.AppImage.zsync"
+  sign-key: None
+  arch: x86_64
```

Comment:
Is AppImage only x86_64, or can it also do i386/i686?

Reply:
Yes, we can make x86, i386, arm64 and armhf builds

Reply:
Could you add all 4 targets for now? We will see how it goes :)

Reply:
I'm not familiar yet with the OpenTTD build systems, and we would have to setup a cross-compile environment for that.

I guess that it's work for a second PR

Reply:
Do you have an example for me how that would look like? Of another project or anything?

Reply:
https://appimage-builder.readthedocs.io/en/latest/examples/bash.html

The complicated part should be cross-compiling OpenTTD, if you already have this then it's simple.
Here is an example on how to build a bash arm64 AppImage in a x86 host system.